### PR TITLE
GLTFExporter: Add forcePOT option on the example

### PR DESCRIPTION
**Pull request by fernandojsg**
Just adding the option that @takahirox implemented on https://github.com/mrdoob/three.js/pull/13424 to the example

### CODE REVIEW

File: examples/misc_exporter_gltf.html [hunk 1]
```diff
@@ -35,6 +35,7 @@
 			<label><input id="option_drawrange" name="visible" type="checkbox" checked="checked"/>Truncate drawRange</label>
 			<label><input id="option_binary" name="visible" type="checkbox">Binary (<code>.glb</code>)</label>
 			<label><input id="option_forceindices" name="visible" type="checkbox">Force indices</label>
+			<label><input id="option_forcepot" name="visible" type="checkbox">Force PTO textures</label>
```
Comment (mrdoob) [inline examples/misc_exporter_gltf.html]: `Force PTO` ☺

Comment (fernandojsg) [reply examples/misc_exporter_gltf.html]: @mrdoob Done! Force PTO is what I need :_D

**Comment by fernandojsg** [comment]
I've just added a small fix to the example by changing the near value for the ortho camera to be >0 as discussed on https://github.com/mrdoob/three.js/issues/13429

**Comment by takahirox** [comment]
A bit off topic, feeling like we should rename `forcePowerOfTwoTexture` with `forcePowerOfTwoTextures` (adding 's').

**Comment by fernandojsg** [comment]
@takahirox yeah good point :+1: just changed it too

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**